src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Models/CommonModels/Utils.cs

```diff
@@ -261,6 +268,10 @@ public static WorkloadType GetPsWorkloadType(string workloadType)
             {
                 return WorkloadType.AzureSQLDatabase;
             }
+            if (workloadType == ServiceClientModel.WorkloadType.AzureFileShare.ToString())
```

Comment:
Check if ToString()is necessary here. In the above section, for BackupManagementType, you don't seem to be using it.